tests/garage/envs/mujoco/randomization/test_dynamics_rand.py

```diff
@@ -1,26 +1,119 @@
 import unittest
 
-from garage.envs.mujoco import SwimmerEnv
+import numpy as np
+
 from garage.envs.mujoco.randomization import Distribution
 from garage.envs.mujoco.randomization import Method
 from garage.envs.mujoco.randomization import randomize
 from garage.envs.mujoco.randomization import Variations
+from garage.envs.mujoco.swimmer_env import SwimmerEnv
 
 
 class TestDynamicsRand(unittest.TestCase):
-    def test_dynamics_rand(self):
+    def test_dynamics_rand_absolute(self):
+        bodyname = 'mid'
+        attribute = 'pos'
+        lower_bound_mult = np.array([0.5, 0.5, 0.5])
+        upper_bound_mult = np.array([1.5, 1.5, 1.5])
+
+        # ensure you have legit bounds for randomization
+        assert (all(lower_bound_mult != 1))
+        assert (all(upper_bound_mult != 1))
+        assert (all(upper_bound_mult > lower_bound_mult))
+
+        env = SwimmerEnv()
+        body_id = env.sim.model._body_name2id[bodyname]  # get ID
+        orig_val = np.array(env.sim.model.body_pos[body_id])
+        print('Original', orig_val)  # get original attribute value
+
         variations = Variations()
-        variations.randomize() \
-                .at_xpath(".//geom[@name='torso']") \
-                .attribute("density") \
-                .with_method(Method.COEFFICIENT) \
-                .sampled_from(Distribution.UNIFORM) \
-                .with_range(0.5, 1.5) \
-                .add()
+        variations.randomize()\
+            .at_xpath(f"//body[@name=\'{bodyname}\']")\
```

Comment:
f-expressions unsupported in Python 3.5

Reply:
What python versions do we officially support?

Reply:
3.5+